game/scripts/npc/abilities/lina_laguna_blade.txt

```diff
@@ -26,7 +26,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "70 60 50 50 50"
+    "AbilityCooldown"                                     "50 45 40 35 30" //OAA
```

Comment:
I am having a deja vu. I think I told you to buff just the cd or just dmg. But you forgot. Put `50` at all levels.

Reply:
Lina has a simillar ultimate to skywrath, Qop, Dark Willow and lion and all heroes heroes have much lower cooldown ultimates (Qop also gets a CDR talent so +1 to her).

Reply:
Did you calculate the dps with damage buff? I think cd should not go below 40.

Reply:
Damage gets very high with kaya yasha. At level 20 with a level 2 kaya yasha and aghs its about 1.8k pure.